src/components/login-view/login-view.jsx

```diff
@@ -2,6 +2,8 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import axios from 'axios';
 
+import { connect } from 'react-redux';

```

Comment:
`connect` is not being used in this file, should it be? if not then this line should be removed